app/views/organizations/show.html.slim

```diff
@@ -1,23 +1,18 @@
-.row
-  .col-sm-4
-    h1
-      = link_to 'Organizations', organizations_path
-      = "/#{params[:id]}"
-    p#notice = notice
+.page-header
+  h1
+    = link_to 'Organizations', organizations_path
+    = "/#{params[:id]}"
 
-    p
-      strong Name:
-      br
-        = @organization.name
-    .form-group
-      = link_to 'Back', organizations_path, class: 'btn btn-default'
-      - if policy(@organization).edit?
-        = link_to 'Edit', edit_organization_path(@organization), class: 'btn btn-default'
-      - if policy(@organization).destroy?
-        = link_to 'Destroy', organization_path(@organization), method: 'delete', data: {confirm: 'Are you sure?'}, class: 'btn btn-danger'
+  p
```

Comment:
@Gataniel please move it outside of `.page-header`. It should be below `.page-header`.